vertx-web-sstore-infinispan/pom.xml

```diff
@@ -55,7 +55,7 @@
     </dependency>
     <dependency>
       <groupId>org.infinispan</groupId>
-      <artifactId>infinispan-client-hotrod</artifactId>
+      <artifactId>infinispan-client-hotrod${infinispan.jakartasuffix}</artifactId>
```

Comment:
Same comment here as for `vertx-infinispan`.

Please run the Maven dependency tree goal to check if there are no duplicates when the jakarta profile is enabled.